pendingTranslations.ftl

```diff
@@ -11,3 +11,27 @@ main-nav-link-faq-tooltip = Frequently asked questions
 
 footer-external-link-faq-label = FAQs
 footer-external-link-faq-tooltip = Frequently asked questions
+
+# Status Pill
+
+status-pill-action-needed = Action Needed
+status-pill-progress = In Progress
+status-pill-fixed = Fixed
+
+# Exposure Card
+
+exposure-card-family-members = Family members
+exposure-card-email = Type
+exposure-card-phone-number = Phone number
+exposure-card-address = Address
+# Variables:
+# $exposure_type is the number of exposures found for a particular type, e.g. 3 found 
+exposure-card-num-found = { $exposure_num } found
```

Comment:
```suggestion
exposure-card-num-found = 
  { $exposure_num ->
     *[other] { $exposure_num } found
  }
```

Technically identical for English, but this would trigger the plural UI in Pontoon (if we end up exposing to other locales).